src/content/developers/docs/consensus-protocols/index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Consensus protocols
```

Comment:
I agree with your comment that we should go with consensus mechanisms so as not to confuse with protocols.